third-party/spicedb/README.md

```diff
@@ -0,0 +1,39 @@
+# @kubernetes-models/spicedb
+
+[SpiceDB](https://authzed.com/docs) models.
+
+## Installation
+
+Install with npm.
+
+```sh
+npm install @kubernetes-models/spicedb
+```
+
+## Usage
+
+```js
+import { SpiceDBCluster } from '../gen/authzed.com/v1alpha1/SpiceDBCluster';
```

Comment:
```suggestion
import { SpiceDBCluster } from '@kubernetes-models/spicedb/authzed.com/v1alpha1/SpiceDBCluster';
```

Reply:
updated in https://github.com/tommy351/kubernetes-models-ts/pull/161/commits/5de2333f40cbb4d1fba6b6c89806ebcaca2d4e0b